src/aig-common/modules/standard/components/ipp-sector/ipp-sector-form/ipp-sector-form.component.ts

```diff
@@ -0,0 +1,76 @@
+import { Component, OnInit } from '@angular/core';
```

Comment:
this component not follow the standard indication of NewUpdateForm.
Fix it

Reply:
This component also was modified as *-new-update-form component.